### PR TITLE
feat(core/admin): add discord webhook logs for admin actions

### PR DESCRIPTION
**Pull request by Konijntje-x**
Added discord webhook logs for admin actions. 

There are logs on:

 Kick
 Ban
 spawnprop
 teleport to marker
 spawn vehicle
 freeze/unfreeze
 revive

### CODE REVIEW

File: modules/__core__/admin/client/module.lua [hunk 1]
```diff
@@ -267,6 +281,7 @@ module.RevivePlayer = function(sourceId)
     if not a then return end
 
     local playerPed = PlayerPedId()
+    local playerName = GetPlayerName(PlayerPedId)
```
Comment (ESXCrackhead) [inline modules/__core__/admin/client/module.lua:284]: PlayerPedId is not declared. 

So this line needs to be fixed.

**Review by ESXCrackhead — CHANGES_REQUESTED** [review]
In addition to the messed up variable,

Rather than requiring players to delete lines throughout the file, we should add a config option in
`esx-reborn/config/default/config.admin.lua` like this:


```
Config.Modules.Admin = {
  deleteVehicleRadius = 5,
  useDiscordLogs = true
}
```

and in the files

```
if Config.Modules.Admin.useDiscordLogs then
  -- LogStuff
end
```

**Comment by ESXCrackhead** [comment]
@Konijntje-x

**Comment by Konijntje-x** [comment]
> ```
> if Config.Modules.Admin.useDiscordLogs then
>   -- LogStuff
> end
> ```

if Config.Modules.Admin.useDiscordLogs == true then
or just Config.Modules.Admin.useDiscordLogs?
@ESXCrackhead

**Comment by ESXCrackhead** [comment]
That's better.

Now the final change is that you need to update the PR title to match conventional commit messages

https://www.conventionalcommits.org/en/v1.0.0/

(look at other commits to see the formatting for conventional commit messages)

I will do it for you this time, but in the future I will not accept PRs that do not follow conventional commit messages properly.

**Review by ESXCrackhead — APPROVED** [review]

**Pull request merged**